src/layout/HeadingContainer.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "@emotion/styled";
+
+export const HeadingContainer = styled.div`
+  height: 2.6rem;
+  margin: .2rem;
+  margin-top: 0rem;
+  border-bottom: 2px solid #f0f0f0;
+`;
```

Comment:
Please,  add new line at the end of tile :)

Reply:
do you mean adding space beneath the title?

Reply:
i guess, you mean a new line in the .tsx file at the end, right?